data/crac/crac-creator-api/src/main/java/com/farao_community/farao/data/crac_creator_api/CracCreationReport.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.farao_community.farao.data.crac_creator_api;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Common methods used in CRAC creation reports
+ *
+ * @author Godelaine de Montmorillon {@literal <godelaine.demontmorillon at rte-france.com>}
+ */
+public final class CracCreationReport {
+    private static final Logger LOGGER = LoggerFactory.getLogger(CracCreationReport.class);
+    private List<String> creationReport;
+
+    public CracCreationReport() {
+        creationReport = new ArrayList<>();
+    }
+
+    public void error(String errorReason) {
+        String message = String.format("[ERROR] %s", errorReason);
+        creationReport.add(message);
+        LOGGER.info(message);
```

Comment:
what do you think about adapting the LOGGER level to the message ?
error -> LOGGER.error()
removed/altered/warn -> LOGGER.warn()
info -> LOGGER.info()

Reply:
changed